explorer/src/Pos/Binary/Explorer.hs

```diff
@@ -6,9 +6,8 @@ import           Universum
 
 import           Pos.Binary              ()
 import           Pos.Binary.Class        (Cons (..), Field (..), deriveSimpleBi)
-import           Pos.Core                (HeaderHash, Timestamp)
+import           Pos.Core                (HeaderHash, Timestamp, TxUndo)
```

Comment:
I suppose `TxUndo` shouldn't be in core, but it's rather small, so I won't be too sad seeing it there.